app/src/main/java/com/stevesoltys/seedvault/ui/storage/StorageRootResolver.kt

```diff
@@ -34,6 +35,15 @@ internal object StorageRootResolver {
                     if (root != null) roots.add(root)
                 }
             }
+            if (UserHandle.myUserId() != UserHandle.USER_SYSTEM) {
+                val c: Context = context.createContextAsUser(UserHandle.SYSTEM, 0)
+                c.contentResolver.query(rootsUri, null, null, null, null)?.use { cursor ->
+                    while (cursor.moveToNext()) {
+                        val root = getStorageRoot(context, authority, cursor)
```

Comment:
`getStorageRoot` root is called with the current user context, not the system context. Is that intentional? Could that have worked in your testing?

Reply:
This context is sent to query package manager for icons. You mentioned you'd prefer using the context of the current user for package manager?

Reply:
Ah ok, it wasn't obvious from the diff that the context is only used to fetch the icon. That's fetching a drawable from the current app in this case, so it probably should use app context, but I'd add a comment explaining why using the app context here and not `c`.

Reply:
Done